tests/color_test.py

```diff
@@ -35,9 +35,16 @@ def test_use_color_no_tty():
         assert use_color('auto') is False
 
 
-def test_use_color_tty():
+def test_use_color_tty_with_color_support():
     with mock.patch.object(sys.stdout, 'isatty', return_value=True):
-        assert use_color('auto') is True
+        with mock.patch('pre_commit.color.terminal_supports_color', True):
```

Comment:
I usually prefer `mock.patch.object` as it's slightly less error prone and has fewer side-effects (`mock.patch` can trigger an import as well as a bunch of `getattr`s)

It doesn't actually matter here so this is fine